### DISCUSSION TITLE
Use Linux not Windows

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [Comfy-Org/ComfyUI#6140] [General]
Just finally ran on Linux and I cannot imagine why I used Windows all this time, setting up the server is done in 5 seconds and doing images went from 5 minutes into 30 seconds with flux, on Windows I use 30-40GB RAM doing my images I just successfully ran on a 16GB RAM stick with 10GB left...

**Comment by ltdrdata** [comment]
I love linux.

**Comment by pkrasicki** [comment]
That's a crazy boost in speed. Are you using AMD GPU?

**Reply by BrechtCorbeel** [reply]
No I just pulled my AMD 7900 XTX GPU for my RTX 4090 because I can't get AMD to work with Comfyui especially through Stability Matrix.

**Comment by jepjoo** [comment]
On Windows it took 5 minutes per Flux image on a 4090? That's just a broken setup somehow or some very unusually high resolution, step count and more.

Basic 1024x1024, 25 steps Flux image on Windows 11 & underclocked 4090 = under 9 seconds (3 it/s). 

Using FP8 fastmode + torch.compile.

**Reply by BrechtCorbeel** [reply]
Yes it is a bit higher and 16GB dev unchained model I couldn't even run the normal setup I'm at 1496 x 998 with 2x upscale and fairly large prompts, steps = 20

Linux with the same workflow did /2 of the regular time with the same actually even worse setup, but I don't think missing some cpu cores matters here.

**Reply by Afaridoon1104** [reply]
Which Linux version is good for ComfyUI?

**Reply by SRStwo** [reply]
> Which Linux version is good for ComfyUI?

Kubuntu 25.10 used to be good. Not sure about the latest version. Disappointed with Tumbleweed now.
